specification/schemas/PredicateValue.json

```diff
@@ -0,0 +1,32 @@
+{
```

Comment:
Should this schema just be called `Value`? As it's not just a value for a predicate, but a more generic value, which in turn can be a predicate.

Reply:
Maybe it's a good idea to split up the different value types into `PredicateValue`, `PathValue`, `ScalarValue` and `ResourceValue` and have the generic `Value` just be:
```
{
  "oneOf": [
    {"$ref": "./ScalarValue"},
    ...
  ]
}